ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1603,7 +1606,7 @@ private enum UnityLayers {
       // Texture the ball.
       navball_changed_ = false;
       if (plotting_frame_selector_.target_override) {
-        set_navball_texture(target_navball_texture_);
+        set_navball_texture(target_navball_texture_!);
```

Comment:
Comment that the textures are set in `OnAwake` (we could write lazy-load properties for them so that things are clearer).